website/versioned_docs/version-0.60/dimensions.md

```diff
@@ -4,6 +4,128 @@ title: Dimensions
 original_id: dimensions
 ---
 
+> [`useWindowDimensions`](useWindowDimensions) is the preffered API for React components. Unlike `Dimensions`, it updates as the window's dimensions update. This works nicely with the React paradigm.
```

Comment:
useWindowDimensions is only available from React Native 0.61, so it might be better to make a separate document of dimensions for version 0.61 🤔

Reply:
Would you please?

Reply:
@rachelnabors done!